rust/rsc/src/bin/rsc/types.rs

```diff
@@ -172,3 +173,27 @@ pub enum ReadJobResponse {
 pub struct GetUploadUrlResponse {
     pub url: String,
 }
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct DashboardStatsOldestJob {
+    pub label: String,
```

Comment:
`label` is an `Option<String>` in the job payload are we storing the empty string in the DB for none?

Reply:
Yeah, `label` is non-optional in the database model so it picks up the default of empty string.

https://github.com/sifive/wake/blob/master/rust/entity/src/job.rs#L32